buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy

```diff
@@ -216,6 +216,9 @@ class IndexGenerator {
             case Category.APPRENTICE:
                 return 'https://micronaut.io/wp-content/uploads/2020/11/Misc.svg'
 
+            case Category.ORACLE_CLOUD:
+                return 'https://micronaut.io/wp-content/uploads/2020/11/Misc.svg' // TODO
```

Comment:
@JasonTypesCodes can you ask the web designers to create a SVG for Oracle Cloud?